recipes/brainglobe-atlasapi/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "brainglobe-atlasapi" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/brainglobe-atlasapi-{{ version }}.tar.gz
+  sha256: d8fc449e415d510933d4b280947a2524f9f7292cb15f8db6d342f5dc4c02220b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=45
+    - wheel
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >=3.9
+    - brainglobe-space >=1.0.0
+    - click
+    - loguru
+    - meshio
+    - numpy
+    - pandas
+    - pyarrow
+    - pymcubes
+    - requests
+    - rich >=9.0.0
+    - simpleitk
+    - tifffile
+    - tqdm >=4.46.1
+    - treelib
+    - vedo
+    - vtk
+    - xmltodict
+
+test:
+  imports:
+    - brainglobe_atlasapi
+  commands:
+    - pip check  # [not win]
```

B/c this is noarch we don't necessarily require Windows to pass. However, I want to check the logs what is going on here to ensure the package will be functional on Windows.

```suggestion
    - pip check
```